otter_welcome_buddy/common/discord_channel.py

```diff
@@ -0,0 +1,25 @@
+import os
+from enum import Enum
+
+from discord.ext.commands import Bot
+from dotenv import load_dotenv
+
+from otter_welcome_buddy.common.utils.types.common import DiscordChannelType
+
+load_dotenv()
+
+
+class DiscordChannel(Enum):
+    """
+    Enum to store Discord channels variables
+    """
+
+    ENGLISH_CHANNEL_ID = int(os.environ["GENERAL_CHANNEL_ID"])
+    NON_ENGLISH_TEACHERS_CHANNEL_ID = int(os.environ["ROOT_CHANNEL_ID"])
+    ENGLISH_CHANNEL_IMAGE = "https://shorturl.at/blqMV"
```

Comment:
Can't you use the image directly from the gh repo?
https://raw.githubusercontent.com/Proyecto-Nutria/proyecto-nutria.github.io/b867badd44de765ced1e20e60eb7a04d73ee1685/public/images/Intro/intro_header.webp

Reply:
Also, thinking twice maybe we can reuse this variable, in different places, can you add it to the constants file and change the name to something like `OTTER_HEADER_IMAGE`?